### DISCUSSION TITLE
I need an help so i really want this chibika replaced quetzal plushie but i dont know how to do it? could you guys tell me how to do it? <3

### DISCUSSION THREAD

**Discussion by LILSALSA123** [Monika-After-Story/MonikaModDev#9251] [QA - tech support + general]
![Screenshot_1](https://user-images.githubusercontent.com/106671273/172016080-9f21b99b-78ef-49ca-80c4-5c6f20df9de9.png)

**Comment by calvin12345678** [comment]
just do the same step when you change it to that chibika. but use the quetzal file/image
